ext/standard/tests/array/array_column_basic.phpt

```diff
@@ -82,14 +82,14 @@ $numericCols = array(
 );
 var_dump(array_column($numericCols, 1));
 var_dump(array_column($numericCols, 1, 0));
-var_dump(array_column($numericCols, 1, 0.123));
+var_dump(array_column($numericCols, 1, 0));
```

Comment:
Dropping tests with float keys in arrays is fine, but I don't think you should be dropping tests where it's a parameter. How array_column deals with a float key is relevant.

Reply:
Indeed, and it's one of the least problematic use cases, are there any other functions where I should revert it?